python-test/features/steps/agentsProvider.py

```diff
@@ -0,0 +1,54 @@
+from behave import given, when, then
+import time
+import functions
+import os
+import docker
+
+email = os.getenv('EMAIL')
+password = os.getenv('PASSWORD')
+
+agentname = os.getenv('AGENT_NAME')
+agentTagKey = os.getenv('TAG_KEY', 'test')
+agentTagValeu = os.getenv('TAG_VALUE', 'true')
+
+
+@given('A valid authentication')
+def get_auth(context):
+    context.token = functions.generate_token(email, password)
+    assert len(context.token) > 0
+
+@when('Create an agent')
+def create_agent(context):
+    context.status_agent, context.agent_data = functions.new_agent(context.token, agentname, agentTagKey, agentTagValeu)
+    assert context.status_agent == 201
+    context.agent_key = context.agent_data['key']
+    context.agent_id = context.agent_data['id']
+    context.agent_channel_id = context.agent_data['channel_id']
+
+@when('Run agent container')
+def run_agent(context):
+    iface = os.getenv('IFACE', 'mock')
+    context.orb_address = os.getenv('ORB_ADDRESS', 'beta.orb.live')
+    context.version_image = os.getenv('AGENT_VERSION')
+    context.agent_image = f"ns1labs/orb-agent:{context.version_image}"
+    var_env ={"ORB_CLOUD_ADDRESS":context.orb_address, "ORB_CLOUD_MQTT_ID":context.agent_id,"ORB_CLOUD_MQTT_CHANNEL_ID":context.agent_channel_id, "ORB_CLOUD_MQTT_KEY":context.agent_key,"PKTVISOR_PCAP_IFACE_DEFAULT":iface}
+    client = docker.from_env()
+    context.agent_container = client.containers.run(context.agent_image, detach=True, network_mode= 'host', environment=var_env)
+    assert len(context.agent_container.id) > 0
+    context.id_terminal =  context.agent_container.id
```

Comment:
We should delete the container after the test is done.

Reply:
I think behave has a canonical way to do this type of cleanup, we should use whatever it is.